src/m365/spo/commands/navigation/navigation-node-set.spec.ts

```diff
@@ -81,7 +81,7 @@ describe(commands.NAVIGATION_NODE_SET, () => {
     };
     const patchStub = sinon.stub(request, 'patch').callsFake(async (opts) => {
       if (opts.url === `${webUrl}/_api/web/navigation/GetNodeById(${id})`) {
-        return '';
```

Comment:
My bad, let's restore this value because this is actually what the API is returning.